utils/src/integ-test/java/uk/gov/moj/sdt/test/utils/DBUnitUtility.java

```diff
@@ -1,635 +1,620 @@
+
+            // If it was found, close it.
+            if (dbConnection != null) {
+                try {
+                    dbConnection.close();
+                } catch (final SQLException e) {
+                    LOGGER.error("Error closing connection for Schema '" + dbConnection.getSchema() + "': " + e);
+                }
+
+                // Remove it from the map.
+                allConnections.remove(schema);
+            }
+        }
+    }
+
+    /**
+     * Drop primary keys.
+     */
+    public static void dropPrimaryKeys() {
+        LOGGER.info("Dropping primary keys");
+        runSQLFile("./../utils/src/integ-test/resources/init/drop_primary_keys.sql");
+    }
+
+    /**
+     * Re-enable primary keys.
+     */
+    public static void enablePrimaryKeys() {
+        LOGGER.info("Enabling primary keys");
+        runSQLFile("./../utils/src/integ-test/resources/init/create_primary_keys.sql");
+
+    }
+
+    /**
+     * Run sql files.
+     *
+     * @param fileName the name of the file to run.
+     */
+    private static void runSQLFile(final String fileName) {
+        try {
+            LOGGER.info("Running " + fileName);
+
+            // Get the connection already setup for DBUNIT.
+            final IDatabaseConnection dbConnection = getConnectionInstance("SDT_OWNER");
+            final Connection connection = dbConnection.getConnection();
+            connection.prepareStatement("SET CONSTRAINTS ALL DEFERRED").execute();
+
+            final FileInputStream inputStream = new FileInputStream(fileName);
+
+            final StringBuilder stringBuilder = new StringBuilder();
+            final InputStreamReader inputStreamReader = new InputStreamReader(inputStream);
+            final BufferedReader reader = new BufferedReader(inputStreamReader);
+            String line = reader.readLine();
+            while (null != line) {
+                stringBuilder.append(' ');
+                stringBuilder.append(line);
+                line = reader.readLine();
+            }
+            final String sqlString = stringBuilder.toString();
+            final Statement stmt = connection.createStatement();
+
+            // Execute the query
+            stmt.executeUpdate(sqlString);
+
+            // Close the result set, statement and the connection
+            stmt.close();
+
+        } catch (final SQLException e) {
+            LOGGER.error("Failure to drop constraints", e);
+        } catch (final FileNotFoundException e) {
+            e.printStackTrace();
+        } catch (final IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Method to disable all constraints and triggers for specific schema prior
+     * to loading data via DBUNIT for a specific integration test.
+     *
+     * @param schemaName name of the schema in which constraints and triggers are to be
+     *                   disabled
+     */
+    public static void prepareForDbunitLoad(final String schemaName) {
```

Comment:
Don't forget to remove this method if it is not needed.

Reply:
keepiing these for now, uncommented it now.
will delete DBUnitUtility.java, once we are happy with the changes done and having working application